### ISSUE TITLE
MdOutlinedSelect is blinking on open in react

### ISSUE THREAD

**Issue by MarcosBustamante** [material-components/material-web#5315]
### What is affected?

Component

### Description

When I click to open the select, the animation apparently is broken, since it is blinking

Video:
https://github.com/material-components/material-web/assets/6501860/56546cc4-1faf-4903-a3f2-f0a59dcc16f7

### Reproduction

I am using the lib with React.

React components:
```
import {createComponent} from '@lit/react';
import {MdOutlinedSelect} from '@material/web/select/outlined-select';
import {MdSelectOption} from '@material/web/select/select-option';
import React from 'react';

export const OutlinedSelect = createComponent({
  tagName: 'md-outlined-select',
  elementClass: MdOutlinedSelect,
  react: React,
  events: {
    onChange: 'change',
    onInput: 'input',
    onClose: 'close',
    onClosed: 'closed',
    onOpen: 'open',
    onOpened: 'opened',
  },
});

export const SelectOption = createComponent({
  tagName: 'md-select-option',
  elementClass: MdSelectOption,
  react: React,
  events: {
    onOpen: 'open',
    onClose: 'close',
  },
});
```

My component:
```
export default function MyComponent(props: Props) {
  return (
    <OutlinedSelect {...props}>
      <SelectOption value="1">
        <div slot="headline">1</div>
      </SelectOption>
     <SelectOption value="2">
        <div slot="headline">2</div>
      </SelectOption>
      <SelectOption value="another">
        <div slot="headline">Another</div>
      </SelectOption>
    </OutlinedSelect>
  );
}
```

### Workaround

If I remove the animation with the flag `quick` the experience is better.

### Is this a regression?

No or unsure. This never worked, or I haven't tried before.

### Affected versions

1.0.0

### Browser/OS/Node environment

macOS v14.0
safari v17.0

**Comment by e111077** [comment]
Should have been fixed in 1.0.1 https://github.com/material-components/material-web/commit/73eb15ebee06d1ac12e94a43d7b59234b8bd48ee

**Comment by e111077** [comment]
If you can still repro this on the latest version, please comment to reopen
